scale/job/views.py

```diff
@@ -1134,6 +1136,42 @@ def _list_v6(self, request):
 
         return self.get_paginated_response(serializer.data)
 
+    def post(self, request):
+        """Creates a new job, places it on the queue, and returns the new job information in JSON form
+
+        :param request: the HTTP POST request
+        :type request: :class:`rest_framework.request.Request`
+        :rtype: :class:`rest_framework.response.Response`
+        :returns: the HTTP response to send back to the user
+        """
+        if request.version != 'v6':
+            raise Http404
+        job_type_id = rest_util.parse_int(request, 'job_type_id')
+        job_data = rest_util.parse_dict(request, 'input', {})
+        job_config = rest_util.parse_dict(request, 'config', {})
+
+        #job_input = DataV6(job_data)
```

Comment:
We're moving to handling json in views and passing objects internally in v6. Need to uncomment this and refactor methods to accept objects instead of json.